package.json

```diff
@@ -14,6 +14,7 @@
     "mocha": "*"
   },
   "scripts": {
+    "pretest": "rsync -av --ignore-existing ./configs/jira_credentials.js.example ./configs/jira_credentials.js",
```

Comment:
@alan-andrade tests were bonking because we don't have any `jira_credentials.js` in the repo anymore, so when it pulls the code and runs the test, it's getting `FileNotFound` errors. no fucking bueno. 

this guy will copy over `jira_credentials` unless it already exists (so that we can run `npm test` locally without fucking up our own `jira_credentials`).